cmd/profilecli/main.go

```diff
@@ -79,6 +79,10 @@ func main() {
 	queryTracerCmd := app.Command("query-tracer", "Analyze query traces.")
 	queryTracerParams := addQueryTracerParams(queryTracerCmd)
 
+	queryBlocksCmd := app.Command("query-blocks", "Query on local/remote blocks")
+	queryBlocksSeriesCmd := queryBlocksCmd.Command("series", "Request series labels on local/remote blocks")
+	queryBlocksSeriesParams := addQueryBlocksSeriesParams(queryBlocksSeriesCmd)
+
```

Comment:
Not directly related to this PR, but if you think it makes sense we can improve the storage config handling (in all subcommands)

I think that we should use the standard client config (that supports all backends). You can register the flags as follows:

```go
	blocksCmd := adminCmd.Command("blocks", "Operate on Grafana Pyroscope's blocks.")
	blocksCmd.Flag("path", "Path to blocks directory").Default(defaultLocalBlockPath).StringVar(&cfg.blocks.path)
	var f flag.FlagSet
	var c client.StorageBackendConfig
	c.RegisterFlags(&f, logger)
	f.VisitAll(func(flag *flag.Flag) {
		blocksCmd.Flag(flag.Name, flag.Usage).Hidden().Default(flag.DefValue).SetValue(flag.Value)
	})
```

This will probably require handling of the default path and fs backend. This way we could create our `Bucket` object for all `blocks` subcommands automatically.